cfgtool

```diff
@@ -285,14 +285,34 @@ Stay safe. Use crypto. Don´t drink and factorize."
 
 }
 
+#http://blog.pragbits.com/it/2015/02/09/slack-notifications-via-curl/
+__notify_slack() {    # format message as a code block ```${msg}```
```

Comment:
Wow! ++1 for introducing slack notifications.

Generally though, please do not post ANY secret keys in repos.
This is low impact since it's in an extra channel, but imagine what would happen if 100k people started using this project. We Slack might get a bit annoyed about us